libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -52,6 +52,17 @@ void copyMissingTags(StructurallyDocumentedAnnotation& _target, set<CallableDecl
 			_target.docTags.emplace(tag, content);
 }
 
+CallableDeclaration const* findBaseCallable(set<CallableDeclaration const*> const& _baseFunctions, int64_t _contractId)
```

Comment:
Why `int`?

Reply:
Because that's the type that `ContractDefinition->id()` returns

Reply:
Indeed. Now I'm confused why `id` is int